api/services.go

```diff
@@ -98,7 +98,7 @@ func serviceTokensPost(s *state.State, r *http.Request) response.Response {
 		return response.BadRequest(err)
 	}
 
-	sh, err := service.NewHandler(s.Name(), req.ClusterAddress, s.OS.StateDir, false, false, types.ServiceType(serviceType))
+	sh, err := service.NewHandler(s.Name(), req.ClusterAddress, s.OS.StateDir, false, false, nil, types.ServiceType(serviceType))
```

Comment:
You've added this config argument to `NewHandler` but it's always set to `nil` in every case. Is there a reason for this or can it be removed?

Reply:
Yes, good catch. I'm pretty sure I can remove this.

Reply:
fixed